server/main.lua

```diff
@@ -97,12 +104,14 @@ local function Transform(source, zone)
 		elseif zone == "TraitementJus" then
 			local itemQuantity = xPlayer.getInventoryItem('raisin').count
 			if itemQuantity <= 0 then
-				TriggerClientEvent('esx:showNotification', source, _U('not_enough_raisin'))
+				xPlayer.showNotification('_U('not_enough_raisin')')
```

Comment:
you make a mistake here with xPlayer.showNotification('_U('not_enough_raisin')') the right syntax is xPlayer.showNotification(_U('not_enough_raisin'))